controllers/managedocs_controller.go

```diff
@@ -1048,6 +1059,16 @@ func (r *ManagedOCSReconciler) reconcilePrometheus() error {
 			},
 			Key: alertRelabelConfigSecretKey,
 		}
+		// Enabling remote write for only provider clusters
```

Comment:
- I believe anything wrt PV(/C) will only be available at consumer clusters
- Maybe we need to enable for consumer as well unless I'm mistaken

Reply:
For now IIRC we are only enabling RHOBS support for provider clusters.